site/en/tutorials/keras/save_and_load.ipynb

```diff
@@ -798,14 +798,14 @@
         "id": "dGXqd4wWJl8O"
       },
       "source": [
-        "Keras saves models by inspecting the architecture. This technique saves everything:\n",
+        "Keras saves models by inspecting their architecture. This technique saves everything:\n",
         "\n",
         "* The weight values\n",
         "* The model's architecture\n",
-        "* The model's training configuration(what you passed to compile)\n",
```

Comment:
How about "compile" -> "`compile()`" (similar to https://www.tensorflow.org/guide/keras/customizing_what_happens_in_fit)? Then, you won't have to state that it's a method.

WDYT?

Reference: [Code in text](https://developers.google.com/style/code-in-text?hl=en#some-specific-items-to-put-in-code-font) (Google developer docs style guide).

```diff
- "* The model's training configuration(what you passed to compile)\n",
[or]
- "* The model's training configuration (what you pass to the .compile method)\n",
+ "* The model's training configuration (what you pass to `compile()`\n",
```

Reply:
sounds good